acceptance/server_test.go

```diff
@@ -94,4 +95,25 @@ func AddHandlers(server *testserver.Server) {
 	server.Handle("POST /api/2.0/workspace/mkdirs", func(r *http.Request) (any, int) {
 		return "{}", http.StatusOK
 	})
+
+	server.Handle("GET /api/2.1/jobs/get", func(r *http.Request) (any, int) {
```

Comment:
This is scoped to the test specifically. We should add this in the `test.toml` file. For an example, refer to `acceptance/workspace/jobs/create`.

Reply:
Ah great, haven't realised it's already available